packages/bits/protractor.conf.js

```diff
@@ -8,8 +8,8 @@ exports.config = {
     // Increasing the all scripts timeout to be much above the default jasmine timeout to make visual tests more robust
     allScriptsTimeout: 100000,
     suites: {
-        e2e: "./spec/**/*.e2e.ts",
-        visual: "./spec/**/*.visual.ts",
+        e2e: "./spec/**/freetype-query-builder.e2e.ts",
```

Comment:
why this line is needed?

Reply:
This was just a temporary thing and I forgot to remove it, but I see that you've already removed it.